uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/api/v3/DocumentTest.php

```diff
@@ -257,4 +264,88 @@ public function testRemindMeDocumentClone() {
     $this->assertEquals(0, $clonedCount['values']);
   }
 
+  /**
+   * Documents related to an assignment were generating a revision every time
+   * they were updated. This caused the cloning functionality to copy each
+   * document multiple times, one for each revision the document had. This test
+   * verifies that even if a document has multiple revisions that meet cloning
+   * criteria, only the latest revision is cloned.
+   */
+  public function testDocumentCloningDoesNotCreateExtras() {
```

Comment:
testDocumentCloningOnlyClonesCurrentRevision() as name would reflect the problem better